packages/app-shortcut/ios/Plugin/AppShortcutPlugin.swift

```diff
@@ -6,13 +6,48 @@ import Capacitor
  * here: https://capacitorjs.com/docs/plugins/ios
  */
 @objc(AppShortcutPlugin)
-public class AppShortcutPlugin: CAPPlugin {
+public class AppShortcutPlugin: CAPPlugin, UIApplicationDelegate {
     private let implementation = AppShortcut()
 
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
-        call.resolve([
-            "value": implementation.echo(value)
-        ])
+    @objc func get(_ call: CAPPluginCall) {
+        let result = implementation.get()
+        call.resolve(result.toJSObject() as! JSObject)
```

Comment:
```suggestion
        implementation?.get(completion: { result, error in
            if let error = error {
                CAPLog.print("[", self.tag, "] ", error)
                call.reject(error.localizedDescription)
                return
            }
            if let result = result?.toJSObject() as? JSObject {
                call.resolve(result)
            }
        })
```

For now, please always use completion callbacks like in the other plugins (for example [here](https://github.com/capawesome-team/capacitor-firebase/blob/1cfcf18b0fd33ce217f6c6ab4b91bc3b9eee7334/packages/firestore/ios/Plugin/FirebaseFirestorePlugin.swift#L35)).